Exercise1(form)/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>jsfiddle.net/201flaviosilva/</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="title">
+        <span id="mindera">Mindera </span>
+        <span id="school"> School</span>
```

Comment:
Por norma quando tens elementos aninhados, é normal usar as `class` e os `id` assim:
```suggestion
        <span id="title-mindera">Mindera </span>
        <span id="title-school"> School</span>
```